packages/react-components/react-table/src/index.ts

```diff
@@ -98,6 +98,7 @@ export {
   useTableSelectionCell_unstable,
   renderTableSelectionCell_unstable,
   tableSelectionCellClassNames,
+  CELL_WIDTH as TableSelectionCellWidth,
```

Comment:
I think every time I've used constants I've always all capsed them, but I think it's not consistent in the repo anyway